srcpkgs/grilo/patches/fix-pc-file.patch

```diff
@@ -0,0 +1,15 @@
+help meson generate a proper datadir and typelibdir in pc file
```

Comment:
This doesn't explain why the patch is necessary. (e.g. why can the pkg-config variables not be used? What goes wrong if they are?)

Reply:
Used your extended description. Thanks!